dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -314,7 +327,8 @@ def run_validation(self, job):
                                                              "_" + (record['fain'] or '-none-') + "_" + \
                                                              (record['uri'] or '-none-')
                         passed_validations, failures, valid = Validator.validate(record, csv_schema,
```

Comment:
Is there any way that Validator.validate() is called that isn't when `file_type` is not  `detached_award`?

Reply:
Yes, it's called for every DABS validation, which is why `required_list` and `type_list` are only populated if it's a FABS submission (`detached_award` type) and in the validate function itself it checks if they exist before altering the error report